.changeset/serious-books-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
+---
+
+Correcting e2e test case ids
```

Comment:
suggestion: I think we can make it better, so that it's clear in changelog what was updated. IMHO, we can use your PR description ;)

```suggestion
Changed e2e test case ids to remove duplicate case ids and fix inconsistencies between repo and Testmo. Thanks to this, our QA reports will be more accurate
```